ambry-api/src/main/java/com.github.ambry/config/ClusterMapConfig.java

```diff
@@ -152,6 +152,13 @@
   @Default(MAX_REPLICAS_ALL_DATACENTERS)
   public final String clusterMapDefaultPartitionClass;
 
+  /**
+   * The current xid for this cluster manager. Any changes beyond this xid will be ignored by the cluster manager.
+   */
+  @Config("clustermap.current.xid")
+  @Default("Long.MAX")
```

Comment:
Correct me if I am wrong, the default value of current xid is better to be `Long.MIN` otherwise, if not specified, both frontend and server will honor any instanceConfig changes.

Reply:
`Long.MIN` should not be the default, as that would cause all changes to be ignored. Setting default as `Long.MAX` preserves existing behavior.